scripts/ci/pipeline-netcore-runtime.yml

```diff
@@ -82,10 +82,16 @@ jobs:
 
   - bash: |
       ./autogen.sh --with-core=only
-      make -j 4
+      make -C mono -j4
```

Comment:
This is redundant, make builds everything

Reply:
@marek-safar I don't know why but `make` from the root folder sometimes hangs (happened for @steveisok too afair)

Reply:
does it still happen? I thought I fixed by removing that recursive target from netcore

Reply:
happened for me 15 minutes ago :(

Reply:
hm.. `build.sh` used to run xunit tests via ./run-step script (which could handle timeouts) now the macOS lane is dead cause `make -C netcore run-tests-corefx` can't do it.

Reply:
make -j <any number> seems to hang all the time for me.  Just plain make does not.

Reply:
could you debug that, I tried 10 times in a row with clean configure with no hang with tip of master